src/main.cc

```diff
@@ -53,7 +53,7 @@ int main() {
         modulator_bank.at(i).run();
 
         // Power Meter
-        power_meter_bank.emplace_back(num_samples, modulator_out_pipes.at(i), power_out_pipes.at(i), timeout);
+        power_meter_bank.emplace_back(num_samples, 1, modulator_out_pipes.at(i), power_out_pipes.at(i), timeout);
```

Comment:
Make this 10 please or break it out to a variable at the top of main.

Reply:
Done